tests/integration/test_send_to_annotate.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from labelbox import UniqueIds
+from labelbox.schema.conflict_resolution_strategy import ConflictResolutionStrategy
+
+
+def test_send_to_annotate_include_annotations(
+        client, configured_batch_project_with_label, project_with_ontology):
+    [source_project, _, data_row, _] = configured_batch_project_with_label
+    [destination_project, _] = project_with_ontology
+
+    try:
+        queues = destination_project.task_queues()
+        initial_labeling_task = next(
+            q for q in queues if q.name == "Initial labeling task")
+        print(initial_labeling_task)
```

Comment:
Were the print statements left here on purpose?

Reply:
No, I need to come back and clean them up. I wanted to first make sure the tests were passing on all python versions